helpers/helpers_test.go

```diff
@@ -357,6 +358,12 @@ func TestParsePrivateKeyPEM(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	testEd25519PEM, _ := ioutil.ReadFile(testPrivateEd25519Key)
```

Comment:
You should check for errors, even in a test file.

Reply:
Skipping this check is a convention in this particular `Test*()`. Shall I change this here and above?

Reply:
If it is, maybe I didn't review it --- I would have said the same thing. I think it would be better to have the error check here.

Reply:
Fixed, here and elsewhere in this file.